code/backend/batch/utilities/helpers/ConfigHelper.py

```diff
@@ -70,13 +70,19 @@ def __init__(self, logging: dict):
 class ConfigHelper:
     @staticmethod
     def get_active_config_or_default():
-        try:
-            blob_client = AzureBlobStorageClient(container_name=CONFIG_CONTAINER_NAME)
-            config = blob_client.download_file("active.json")
-            config = Config(json.loads(config))
-        except Exception:
-            print("Returning default config")
-            config = ConfigHelper.get_default_config()
+        env_helper = EnvHelper()
+        config = ConfigHelper.get_default_config()
+
+        if env_helper.LOAD_CONFIG_FROM_BLOB_STORAGE:
```

Comment:
![lgtm](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZ2s5MnVzODNmd2NuazMzODczMWJvbWZ1ODdjemRmamF1djdycDM2byZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/Pn0gCV1oBkr3OUNc21/giphy.gif) - this has been a problematic area to test for lots of things - so thanks!

Reply:
It caused the tests to be suuuuuper slow